satellite/contact/endpoint.go

```diff
@@ -49,23 +47,32 @@ func (endpoint *Endpoint) CheckIn(ctx context.Context, req *pb.CheckInRequest) (
 
 	peerID, err := identity.PeerIdentityFromContext(ctx)
 	if err != nil {
+		endpoint.log.Info("failed to get node ID from context", zap.String("node address", req.Address), zap.String("error", err.Error()))
 		return nil, rpcstatus.Error(rpcstatus.Unknown, errCheckInIdentity.New("failed to get ID from context: %v", err).Error())
 	}
 	nodeID := peerID.ID
 
 	err = endpoint.service.peerIDs.Set(ctx, nodeID, peerID)
 	if err != nil {
-		return nil, rpcstatus.Error(rpcstatus.Unknown, errCheckInIdentity.New("failed to get ID from context: %v", err).Error())
+		endpoint.log.Info("failed to add peer identity entry for ID", zap.String("node address", req.Address), zap.String("node ID", nodeID.String()), zap.String("error", err.Error()))
```

Comment:
If you use `zap.Stringer("node ID", nodeID)` instead of `zap.String("node ID", nodeID.String())`, it will avoid calling `nodeID.String()` when we're not going to log the message anyway, and it's a bit easier to read. Not a showstopper, just a suggestion